std.go

```diff
@@ -47,6 +51,29 @@ func (l *stdErrLogger) Errorf(_ context.Context, format string, v ...any) {
 	stdf("ERROR", format, v...)
 }
 
+// AddRequestAttribute adds an attribute (key, value) for the parent request log
+// If the key already exists, its value is overwritten
+func (l *stdErrLogger) AddRequestAttribute(key string, value any) error {
+	l.mu.Lock()
+	defer l.mu.Unlock()
+	if l.attributes == nil {
+		l.attributes = make(map[string]any)
+	}
```

Comment:
I think we should establish a constructor now that we have reason to avoid this initialization check.

Reply:
There will be no need for this after `attributes` have been removed.